File: Avaliacao1/dicionario_equipes.py
equipes = {}

while True:
    nome = input('Nome da equipe: ').upper()
    cidade = input('Cidade da equipe: ').upper()

    if nome not in equipes.keys():
        equipes['nome'] = cidade
    else:
        print('Equipe ja cadastrada...')
    opcao = int(input('1 para continuar '))

    if opcao != 1:
        break

for chave, valor in equipes.items():
    print(f'Chave {chave}, Valor: {valor}')

File: Avaliacoes/Avaliacao1/Java/DicionarioEquipes.java
package Avaliacoes.Avaliacao1.Java;

import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class DicionarioEquipes {
    public static void main(String[] args) {
        //nome, cidade
        Map<String, String> equipes =  new HashMap<String, String>();
        
        String nome, cidade, opcao;
        Scanner teclado = new Scanner(System.in);

        do {
            System.out.println("Nome equipe: ");
            nome = teclado.nextLine().toUpperCase();
            System.out.println("Nome cidade: ");
            cidade = teclado.nextLine().toUpperCase();

            if (!equipes.containsKey(nome)) {
                equipes.put(nome, cidade);
            }else{
                System.out.println("Equipe ja cadastrada");
            }
            

            System.out.println("1 -  continua, Outra tecla encerra");
            opcao = teclado.nextLine();

            if (!opcao.equals("1")) {
                break;
            }

        } while (true);

        for (Map.Entry<String, String> equipe : equipes.entrySet()) {
            System.out.println(equipe.getKey() + ": " + equipe.getValue());
        }

        teclado.close();

    }
}
